tools/walletextension/main/cli.go

```diff
@@ -49,5 +54,6 @@ func parseCLIArgs() walletextension.Config {
 		NodeRPCHTTPAddress:      fmt.Sprintf("%s:%d", *nodeHost, *nodeHTTPPort),
 		NodeRPCWebsocketAddress: fmt.Sprintf("%s:%d", *nodeHost, *nodeWebsocketPort),
 		LogPath:                 *logPath,
+		PersistencePathOverride: *persistencePath,
```

Comment:
this is not wired in yet. Is the intention for his PR to just create the plumbing?

Reply:
It is wired in actually, as this value is passed into `func NewPersistence(hostAddr string, persistenceFilePath string) *Persistence` and results in the file path used for persistence being either the default (in users home space), or the value as passed in on the cli (I've tested the change and currently using in my tests where I set the persistence path name to the output directory of the test being executed)

Reply:
ah, yes. it was used in tests.